### DISCUSSION TITLE
how does related items works ?

### DISCUSSION THREAD

**Discussion by web54** [templaza/astroid-framework#830] [Q&A] [answered]
Hello,
In my new site I have related articles displaid , but not in all catégories, and I do not find how joomla choose the related items to display
Can someone give me some tips about it ?

**Comment by sonvnn** [comment, marked as answer]
Go to your article -> keywords
Input same keyword in articles. They will display in related articles section

**Comment by web54** [comment]
thanks a lot, that works
